### ISSUE TITLE
function vm_is_running is not working

### ISSUE THREAD

**Issue by truongquocloc** [pyclashbot/pymemuc#148]
function vm_is_running is not working

i fix it: 

```python
def vm_is_running(self: "PyMemuc", vm_index: int = 0) -> bool:
    """Check if a VM is running

    :param vm_index: VM index. Defaults to 0.
    :type vm_index: int, optional
    :return: True if the VM is running, False otherwise
    :rtype: bool
    """
    _, output = self.memuc_run(["-i", str(vm_index), "isvmrunning"])
    if 'Not Running' in output: return False
    else: return True
```

**Comment by martinmiglio** [comment]
You are welcome to make a pull request! Let me know if you need help.

**Comment by martinmiglio** [comment]
Released as [v0.6.0](https://pypi.org/project/pymemuc/0.6.0/)
